count-things/index.js

```diff
@@ -0,0 +1,37 @@
+import { writeFile } from 'fs/promises';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import jsYaml from 'js-yaml';
+
+import { javascriptAnalysis } from './javascript/analysis.js';
+
+import { processArgs } from '../lib/process-args.js';
```

Comment:
is there a good reason to roll our own utility script for this vs using something like `commander` (or other)?

Reply:
commander is a big thing for making full interactive CLI apps.  we just need to process script arguments.  

there are libraries for that, but this was quicker for me than searching for the library.  not opposed to switching it out later